photon_stream/io/jsonl.py

```diff
@@ -116,10 +116,13 @@ def __init__(self, fin):
     def __iter__(self):
         return self
 
+    def __readline(self):
+        return self.fin.__next__().strip().rstrip(',') # raises StopIteration
+
     def __next__(self):
-        line = self.fin.readline().strip().rstrip(',')
-        if not line:
-            raise StopIteration
+        line = self.__readline()
+        while not line: # skip empty lines
```

Comment:
This might cause a problem when one is reading from a stream which intentionally creates empty lines to signal the end of the jsonl-file. But I think the odds for this are low.